core/capabilities/ccip/ccip_integration_tests/helpers.go

```diff
@@ -406,7 +406,6 @@ func (h *homeChain) AddNodes(
 	p2pIDs [][32]byte,
 	capabilityIDs [][32]byte,
 ) {
-	// Need to sort, otherwise _checkIsValidUniqueSubset onChain will fail
 	sortP2PIDS(p2pIDs)
```

Comment:
Pretty sure this and the associated function def'n can be deleted, sorting is no longer a requirement

Reply:
Anyway, not required to do in this PR ;-)

Reply:
I think this whole directory could probably deleted no? Thats what I had in mind with CCIP-3049

Reply:
If these helpers aren't used anywhere and these tests are not needed, yes.